catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsSource.java

```diff
@@ -801,6 +806,10 @@ protected GetFeatureType buildGetFeatureRequest(Query query) throws UnsupportedQ
                             .getLocalPart();
                 }
 
+                if(StringUtils.isNotBlank(srsName)) {
```

Comment:
Should we use `isNotEmpty()` here?  It seems that the SRS name shouldn't be set for an empty string either